buildscripts/buildRelease.sh

```diff
@@ -253,6 +257,17 @@ else
    exit 1
 fi
 
+# Bump up the version in the NPM package.json
+sed -i -e s/"\"version\": \"[0-9]\.[0-9]\.[0-9].*\","/"\"version\": \"$NextVersion\","/ "src/package.json"
```

Comment:
Please use jq for editing and readin JSON (including the line below).

https://stedolan.github.io/jq/